pkg/aggregationapiserver/apiserver.go

```diff
@@ -0,0 +1,114 @@
+package aggregationapiserver
```

Comment:
Are we using [`apiserver-runtime`](https://github.com/kubernetes-sigs/apiserver-runtime) to help us build the AA?

Reply:
Without using `apiserver-runtime` to build the AA.  Directly use `k8s apiserver` just as https://github.com/kubernetes/sample-apiserver.